NBA-Stats-Dashboard/src/components/ComparisonChart.jsx

```diff
@@ -0,0 +1,16 @@
+import { Chart } from 'react-google-charts';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
Why did you disable the eslint error? It would be better to add the prop-type like I did in Dropdown.jsx for the props.

Reply:
Fixed! You are correct! Prop validation is important.